src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Streamline error management with New Relic errors inbox mobile grouping'
+summary: 'Automatically group thousands of error events into common patterns for faster triage of mobile user issues'
+releaseDate: '2023-12-21'
+learnMoreLink: 'https://docs.newrelic.com/docs/errors-inbox/errors-inbox/#how-groups-work'
+---
+
+New Relic errors inbox now lets you automatically group mobile error events into common patterns, reducing noise, allowing you to focus on identifying and resolving the issues with the highest user impact before they hurt the business. With New Relic errors inbox mobile grouping now you can:
+* **Identify high-impact errors faster:** Automatically identify patterns across thousands of individual errors and turn them into meaningful groups, so you can zero in on the most important issue and priorities. 
+* **Analyze more errors with less effort:** Improve user experience for apps with high-volume mobile engagement while staying under cardinality limits so you can capture and examine the errors that matter most
+* **Improve error triage and issue ownership:** Streamline root cause analysis with better insights into mobile user activity to expand full stack coverage with mobile error grouping is part of the New Relic all-in-one observability platform.
```

Comment:
@ken-barrette I'm not quite following this bullet point. Could we split it into two sentences with something like this:

```
* **Improve error triage and issue ownership:** Streamline root cause analysis with better insights into mobile user activity.  This expands full stack coverage with mobile error grouping as part of the New Relic all-in-one observability platform.
```

Reply:
Done